src/main/webapp/item-page.html

```diff
@@ -14,7 +14,8 @@
     />
 
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
-    <script src="script.js"></script>
+    <script src="dashboard-script.js"></script>
+     <script src="script.js"></script>
```

Comment:
I don't think script.js exists anymore

Reply:
I think it does, the two scripts have different code in my local repo. I'm not sure if I'm still using anything from the original script though.

Reply:
Yeah script.js doesn't exist anymore, you could update your local master and then rebase this current branch to see the new changes.

Reply:
Okay got it